server/plugin.ts

```diff
@@ -110,6 +111,15 @@ export class OpendistroSecurityPlugin
         this.logger
       );
       core.http.registerAuth(auth.authHandler);
+    } else if (config.auth.type == 'jwt') {
```

Comment:
Is it better to have an enum for all the auth types?

Reply:
agreed, will refactor later in a separate PR